main/performance.cpp

```diff
@@ -315,13 +315,13 @@ void Performance::set_navigation_process_time(double p_pt) {
 }
 
 void Performance::add_custom_monitor(const StringName &p_id, const Callable &p_callable, const Vector<Variant> &p_args) {
-	ERR_FAIL_COND_MSG(has_custom_monitor(p_id), "Custom monitor with id '" + String(p_id) + "' already exists.");
+	ERR_FAIL_COND_MSG(has_custom_monitor(p_id), vformat("Custom monitor with id '%s' already exists.", String(p_id)));
```

Comment:
Explicitly casting to `String` is not needed for `vformat()` I think.

Reply:
I think so too, as with the other batch I kept all the other details as they were, this batch is relatively small so will change them here and check, wanted to avoid any potential regression from this as some potential errors might not show up unless the error is actually triggered